metricbeat/module/kibana/stats/stats.go

```diff
@@ -152,39 +164,25 @@ func (m *MetricSet) fetchStats(r mb.ReporterV2, now time.Time) {
 
 	content, err := m.statsHTTP.FetchContent()
 	if err != nil {
-		elastic.ReportAndLogError(err, r, m.Log)
-		return
+		return err
 	}
 
 	if m.XPackEnabled {
 		intervalMs := m.calculateIntervalMs()
-		err = eventMappingStatsXPack(r, intervalMs, now, content)
-		if err != nil {
-			m.Log.Error(err)
-			return
-		}
+		return eventMappingStatsXPack(r, intervalMs, now, content)
 	} else {
-		err = eventMapping(r, content)
-		if err != nil {
-			elastic.ReportAndLogError(err, r, m.Log)
-			return
-		}
+		return eventMapping(r, content)
```

Comment:
This should stay as before as we don't have the new framework in 6.8?

Reply:
Note the name of this function. It's a function that is specific to this metricset, not part of the `mb.MetricSet` interface.